samples/matter/light_switch/sample.yaml

```diff
@@ -95,3 +95,20 @@ tests:
     tags:
       - sysbuild
       - ci_samples_matter
+  sample.matter.light_switch.release.power_consumption:
```

Comment:
if we have door lock, maybe we should remove the configuration for the light switch (since door lock acts as SED in Thread Network)